frontend/src/scenes/dashboard/dashboardLogic.ts

```diff
@@ -552,6 +564,15 @@ export const dashboardLogic = kea<dashboardLogicType>({
                 },
             ],
         ],
+        shortIdToId: [
+            (s) => [s.items],
+            (items) => {
+                return (items ?? []).reduce((acc, curr) => {
+                    acc[curr.short_id] = Number(curr.id)
+                    return acc
+                }, {} as Record<InsightShortId, number>)
+            },
```

Comment:
Could this make the typing more strict?

```suggestion
            (items): Record<InsightShortId, number> => {
                return (items ?? []).reduce((acc, curr) => {
                    acc[curr.short_id] = Number(curr.id)
                    return acc
                }, {})
            },
```